.devcontainer/devcontainer.json

```diff
@@ -32,6 +32,12 @@
 			"version": "os-provided"
 		}
 	},
+
+	// Uncomment this if GPU support it is required
```

Comment:
NIT: 
probably we could remove "it" also in the previous string.

Reply:
Sure thing, I'll also update the top comment to reflect the same:

"// Uncomment this if GPU support is required"